build/ac-macros/fla_check_enable_lapack2flash.m4

```diff
@@ -0,0 +1,61 @@
+AC_DEFUN([FLA_CHECK_ENABLE_LAPACK2FLASH],
+[
+	dnl Tell the user we're checking whether to enable the option.
+	AC_MSG_CHECKING([whether user requested the lapack2flash compatibility layer])
+	
+	dnl Determine whether the user gave the --enable-<option> or
+	dnl --disable-<option>. If so, then run the first snippet of code;
+	dnl otherwise, run the second code block.
+	AC_ARG_ENABLE([lapack2flash],
+	              AS_HELP_STRING([--enable-lapack2flash],[Compile and build into libflame a compatibility layer that maps LAPACK invocations to their corresponding FLASH/C implementations. Note that erroneous input parameters are reported according to libflame conventions, NOT LAPACK conventions. That is, if libflame error checking is disabled, no error checking is performed, and if erroneous input parameters are detected, the library aborts. Also, if this option is enabled, then external-lapack-for-subproblems MUST be disabled. (Disabled by default.)]),
```

Comment:
add restrictions metioned above here in doc

Reply:
Fixed: https://github.com/iotamudelta/libflame/pull/2/commits/bd8b490c69720ca6210141fe13ac0703afc20b18

Reply:
I believe this fulfills the restrictions?  https://github.com/iotamudelta/libflame/pull/2/commits/bd8b490c69720ca6210141fe13ac0703afc20b18

Reply:
Actually, it does not, will address.

Reply:
Fixed: https://github.com/iotamudelta/libflame/pull/2/commits/0382f020b4917275810e797dff4174e6166747b8